workflow/rules/deepvariant.smk

```diff
@@ -48,3 +48,21 @@ rule deepvariant:
         "{params.output_gvcf} "
         "--intermediate_results_dir {params.int_res} "
         "--num_shards {threads} ) &> {log}"
+
+
+use rule deepvariant as deepvariant_pacbio with:
+    input:
+        bam="alignment/pbmm2_align/{sample}_{type}.bam",
+        bai="alignment/pbmm2_align/{sample}_{type}.bam.bai",
+        ref=config.get("reference", {}).get("fasta", ""),
+    output:
+        vcf=temp("snv_indels/deepvariant/{sample}_{type}_{chr}.vcf.gz"),
+        gvcf=temp("snv_indels/deepvariant/{sample}_{type}_{chr}.g.vcf.gz")
+        if config.get("deepvariant", {}).get("output_gvcf", False)
+        else [],
+    params:
+        model_type=config.get("deepvariant", {}).get("model_type", "PACBIO"),
+        output_gvcf=lambda wildcards: get_gvcf_output(wildcards, "deepvariant"),
+        int_res=lambda wildcards: f"snv_indels/deepvariant/{wildcards.sample}_{wildcards.type}_{wildcards.chr}",
+        regions=lambda wildcards: f" --regions {wildcards.chr} ",
+        extra=config.get("deepvariant", {}).get("extra", ""),
```

Comment:
Should the parameters be the same as for deepvariant?

Reply:
The difference in this rule is the model_type which here is defaults to 'PACBIO' if the model_type is not set in the config